src/main/java/edu/harvard/iq/dataverse/TemplatePage.java

```diff
@@ -145,7 +142,7 @@ public String init() {
             template = new Template(this.dataverse);
             TermsOfUseAndAccess terms = new TermsOfUseAndAccess();
             terms.setTemplate(template);
-            terms.setLicense(TermsOfUseAndAccess.License.CC0);
+            terms.setLicense(licenseServiceBean.getDefault());
```

Comment:
@qqmyers choosing a default license when creating a new template is the right behavior I think. And I think we agreed that setting the default license in the TermsOfUseAndAcces constructor is not possible due to database link via `@Entity` class

Reply:
FWIW: I think the preferred pattern would be to create a TermsofUseAndAccessServiceBean with a createNewTerms() method that can handle this instead of calling the entity constructor. Since there are relatively few places where you have to pass things around now (since we don't have to deal with the null terms on existing versions), I taked with IQSS about this and I think it's up to you whether you'd want to do that refactoring versus just passing the service around.